tests/phpunit/tests/ajax/ReplytoComment.php

```diff
@@ -264,7 +264,7 @@ public function test_pre_comments_approved() {
 	/**
 	 * Blocks comments from being saved on 'pre_comment_approved', by returning WP_Error.
 	 */
-	function _pre_comment_approved_filter( $approved, $commentdata ) {
+	public function _pre_comment_approved_filter( $approved, $commentdata ) {
```

Comment:
@costdev This one is also a filter callback. To keep it consistent, I'd suggest:

```suggestion
	public function filter_pre_comment_approved_filter( $approved, $commentdata ) {
```

but also changing where it's registered to the filter.